src/java/htsjdk/samtools/BamIndexValidator.java

```diff
@@ -84,4 +84,72 @@ public static int exhaustivelyTestIndex(final SamReader reader) { // throws Exce
         return 0;
     }
 
+    /**
+     * A less time-consuming index validation that only looks at the first and last references in the index
+     *
+     * @param reader
+     * @return # of chunks examined, or 9 if there is no browseable index for the reader
+     */
+    public static int lessExhaustivelyTestIndex(final SamReader reader) {
+        // look at all chunk offsets in a linear index to make sure they are valid
+
+        if (reader.indexing().hasBrowseableIndex()) {
+
+            // content is from an existing bai file
+            final CachingBAMFileIndex existingIndex = (CachingBAMFileIndex) reader.indexing().getBrowseableIndex();
+            final int n_ref = existingIndex.getNumberOfReferences();
+
+
+            int chunkCount = 0;
+            int indexCount = 0;
+            for (int i = 0; i < n_ref; i++) {
+
+                // Whether we are validating this reference
+                final boolean validateThisReference = (i == 0 || i == (n_ref-1));
+
+                final BAMIndexContent content = existingIndex.getQueryResults(i);

```

Comment:
Needed in order to have the chunkCount incremented properly (else branch), as it's reported if an exception is thrown.  @tfenne @eitanbanks , is having this chunk count really useful?  Can we skip it in the light-validation?